gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinPortalNavigationEditNavigationTestIT.java

```diff
@@ -51,6 +53,7 @@ public void setupBeforeMethod() {
    * <li>Test Case Name:Edit node.</li>
    */
   @Test
+  @Disabled
```

Comment:
yes it is due to a bug in PLF but we didn't create an issue in JIRA until we find the cause of the bug